amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -753,6 +753,7 @@ enum ColonyActionType {
   An action related to creating expenditure (dvanced payment) 
   """
   CREATE_EXPENDITURE
+  TEMP_ADVANCED_PAYMENT
```

Comment:
Looks like you've pulled my changes from #1969 - that means `TEMP_ADVANCED_PAYMENT` isn't needed anymore